cypress/e2e/po/pages/diagnostics.po.ts

```diff
@@ -0,0 +1,30 @@
+import PagePo from '@/cypress/e2e/po/pages/page.po';
+import AsyncButtonPo from '@/cypress/e2e/po/components/async-button.po';
+
+/**
+ * Diagnostics page
+ */
+export default class DiagnosticsPagePo extends PagePo {
+  static url = '/diagnostic'
+
+  constructor(pageUrl = DiagnosticsPagePo.url) {
+    DiagnosticsPagePo.url = pageUrl;
+    super(DiagnosticsPagePo.url);
+  }
+
+  downloadDiagnosticsPackageClick() {
+    const btn = new AsyncButtonPo('[data-testid="diagnostics-download-diagnostic-package"]');
+
+    btn.click(true);
+
+    return this;
```

Comment:
this line and from other fn should be `return this`.

Reply:
fixed